pythonmarketo/client.py

```diff
@@ -236,7 +237,29 @@ def create_lead(self, lookupField, lookupValue, values):
             ]
         }
         return self.post(data)
+    
```

Comment:
Here you go...
- Sandip
 


     On Tuesday, May 19, 2015 10:19 AM, Arunim Samat <notifications@github.com> wrote:
   

 In pythonmarketo/client.py:> @@ -236,7 +237,29 @@ def create_lead(self, lookupField, lookupValue, values):
>              ]
>          }
>          return self.post(data)
> +    
Do you have any test for this method so that I can run it before merging the pulled changes?—
Reply to this email directly or view it on GitHub.